docs/user_guide/5_changelog.md

```diff
@@ -95,7 +116,7 @@
 * Showing allowed options for general info template
 * Re-write data visualizer
 
-### Version 1.0.0 (previous noetic-release)
+### Version 1.0.0
```

Comment:
It will be good to keep this comment for previous releases.

Reply:
should this comment be to every version that was used for release or only the 1 before the newest release?

Reply:
It should be there for all of them

Reply:
Added previous noetic-release information now